styleguide/source/_patterns/01-atoms/links.json

```diff
@@ -0,0 +1,17 @@
+{
+  "links": [
+    {
+      "title": "Default Link Style",
+      "href": "#",
+      "text": "Default Link Style"
+    },
+    {
+      "title": "Link to something",
```

Comment:
I don't know if you want all the variants in the list or not, but you could add Link - Black variant in the menu by creating a link~black.json and link-black.md

Reply:
I think I want them all on this list. I'm trying to follow what the AMA SG team is doing, and this is what they were doing for buttons, links, headings, etc. Unless you feel strongly otherwise.